README.md

```diff
@@ -1,24 +1,169 @@
+  </ul>
+</details>
 
-* Database creation
+<details>
+  <summary>Test</summary>
+  <ul>
+    <li><a href="https://rspec.com/">RSPEC</a></li>
+  </ul>
+</details>
 
-* Database initialization
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-* How to run the test suite
+<!-- Features -->
 
-* Services (job queues, cache servers, search engines, etc.)
+### Key Features <a name="key-features"></a>
 
-* Deployment instructions
+- **[Add category]** - Create a category.
+- **[Display all Recategories]** - Display all of users categories.
+- **[Create deal ]** - Create a deals for category.
+- **[Display deal]** - Display all of deals category.
 
-* ...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Version <a name="live-demo"></a>
+
+- [Comming soon]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Presentation -->
+
+## 🚀 Presenation video <a name="live-demo"></a>
+
+- [Video](https://drive.google.com/file/d/1Iu-ubjm5I7dyXyw5yfJep3XAd3SjsGuj/view?usp=sharing)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: Visual Studio Code and Installed Rails.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```bash command
+$ git clone $ git clone https://github.com/iambenkis/Budget-app-with-rails.git
+```
+
+### Install
+
+Install this project with:
+
+```bash command
+you@your-Pc-name:~$ cd <budget-app>
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```bash command
+` $ rails server`
+```
+
+### Run tests
+
+To run tests, run the following command:
+```bash command
+ $ rspec
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Benjamin Kisenge**
+
+- GitHub: [@iambenkis](https://github.com/iambenkis)
+- LinkedIn: [Benjamin Kisenge ](https://www.linkedin.com/in/benjamin-kisenge/)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Add some tests]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give as a star! ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- Microverse
```

Comment:
- [ ] According to project requirements, you should give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).

Project requirement:
<img width="764" alt="image" src="https://user-images.githubusercontent.com/77190203/211314209-c5414f3e-a2e4-4e7b-b97b-f6db3755e924.png">